inst/WORDLIST

```diff
@@ -0,0 +1,3 @@
+ADaM
+Biologics
+CDISC
```

Comment:
Can we just bring over the entire list from admiral.  It is very Pharma specific.

Reply:
Ah yes, will do.